mantidproject/mslice
# Set Window Title on Plot Figure

It makes sense to set the window title to be the same as the figure title, so I've added a setWindowTitle call to the title setter.
Unfortunately this means that the window title will be 'Figure x' as before until the title is modified in plot options, so will not match the figure title on launch. Fixing this seems non-trivial since the figure title isn't decided until sometime after PlotFigureManager's init method.

**To test:**
-Open any plot and go to plot options
-Change figure title
-new figure title should also be displayed as the window title (replacing 'Figure 1')

Fixes #107
